run_all_mac.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+export hse=$(pwd)
+#newtab='tell application "System Events" to keystroke "t" using command down'
+echo $hse
+
+export NOTIFY_API_KEY="${NOTIFY_API_KEY:-test-key}"
+export GOOGLE_ANALYTICS_TRACKING_ID="test-key"
+echo 'Notify key being used is: ' $NOTIFY_API_KEY
+
+#osascript -e 'tell application "Terminal"' -e 'activate' -e 'to do script cd $hse/htbhf-applicant-web-ui; git status; npm start; pause.sh' -e 'end tell'
+osascript -e "tell application \"Terminal\" to do script \"cd $hse/htbhf-applicant-web-ui; git status; npm start; pause.sh\""
```

Comment:
Can you modify this so that the root directory is specified once and can easily be changed for use by someone else without having to modify each of these lines?